tests/dummy/app/templates/components/print-test-attributes.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="attr">{{data-test-attribute}}</div>
+<div class="second">{{data-test-second}}</div>
+<div class="non-test">{{data-non-test}}</div>
+<div class="data-test">{{data-test}}</div>
```

Comment:
I think the class names in here should be named consistently to reflect the corresponding attribute. I would suggest the class name to be the same name as the attribute, so we have `.data-test-attribute`, `.data-test-second`, `.data-non-test` and `.data-test`.